config.json

```diff
@@ -186,6 +186,18 @@
       "values": [ "debug", "release", "checked" ],
       "defaultValue": "debug"
     },
+    "EnableProfileGuidedOptimization": {
+      "description": "Enables IBC profile optimizations if profile data are available.",
+      "valueType": "property",
+      "values": [ true, false ],
+      "defaultValue": false
+    },
+    "OptimizationDataDir": {
+      "description": "Sets the path where the build should look for IBC profile data.",
+      "valueType": "property",
+      "values": [],
+      "defaultValue": ""
+    },
```

Comment:
I'm not opposed to this if it makes things easier for you, but I haven't been updating/using config.jsons for new features so I don't accrue maintenance costs. The plan is to (somehow) make these common properties shared: https://github.com/dotnet/buildtools/issues/1085.